servers/zms/schema/zms_server.sql

```diff
@@ -10,16 +10,19 @@ SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='TRADITIONAL,ALLOW_INVALID_DATES';
 -- -----------------------------------------------------
 -- Schema zms_server
 -- -----------------------------------------------------
+DROP SCHEMA IF EXISTS `zms_server` ;
 
 -- -----------------------------------------------------
 -- Schema zms_server
 -- -----------------------------------------------------
-CREATE SCHEMA IF NOT EXISTS `zms_server` DEFAULT CHARACTER SET utf8 COLLATE utf8_general_ci ;
+CREATE SCHEMA IF NOT EXISTS `zms_server` DEFAULT CHARACTER SET utf8 ;
 USE `zms_server` ;
 
 -- -----------------------------------------------------
 -- Table `zms_server`.`domain`
 -- -----------------------------------------------------
+DROP TABLE IF EXISTS `zms_server`.`domain` ;
```

Comment:
I selected the option to drop schema/tables, thinking most folks using this script would want to start from a clean slate. 

I re-generated the script and removed the drop statementns.